CanvasPlusPlayground/Features/Files/CourseFileService.swift

```diff
@@ -101,6 +112,11 @@ struct CourseFileService {
         }
     }
     
+    /// Opens finder tab at specific directory URL
+    static func showInFinder(fileURL: URL = documentsURL) {
```

Comment:
I'm thinking it goes _inside_ the root directory, instead of opening documents - because documents can be very cluttered for a lot of people (or it should at least select the folder)